testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -288,3 +428,45 @@ pub fn scale_sts_replica(sts_name: &str, replica_num: u64) -> Result<()> {
 
     Ok(())
 }
+
+// XXX: quick helpers around helm operation on the default namespace
+fn get_helm_status(helm_release_name: &str) -> Result<Value> {
```

Comment:
you can probably remove this if we simply do not allow deployment into the `default` namespace, but rather just `forge-` named namespaces.

Reply:
we still need this as long as we are reusing values, see my other comment that explains why